Can't compile syn dependency due to missing feature

```toml
[package]
name = "gen-example"
version = "0.1.0"
authors = ["Michael Lamparski <diagonaldevice@gmail.com>"]
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
genawaiter = "0.99.0"
```

```
C:/Users/diago/.cargo/bin/cargo.exe run --color=always
   Compiling genawaiter-proc-macro v0.99.0
error[E0432]: unresolved imports `syn::FnArg`, `syn::ItemFn`
  --> C:\Users\diago\.cargo\registry\src\github.com-1ecc6299db9ec823\genawaiter-proc-macro-0.99.0\src\lib.rs:20:5
   |
20 |     FnArg,
   |     ^^^^^ no `FnArg` in the root
21 |     Ident,
22 |     ItemFn,
   |     ^^^^^^ no `ItemFn` in the root
```

It seems to me that `genawaiter-proc-macro` requires the `"full"` feature of `syn`, but does not enable it.  (How is this not failing on CI testing?)

--- a/genawaiter-proc-macro/Cargo.toml
+++ b/genawaiter-proc-macro/Cargo.toml
@@ -16,7 +16,7 @@ proc-macro = true
 [dependencies]
 quote = "1.0"
 proc-macro2 = "1.0"
-syn = { version = "1.0", features = ["visit-mut"] }
+syn = { version = "1.0", features = ["visit-mut", "full"] }
 proc-macro-error = "0.4"
 proc-macro-hack = "0.5"
 

